Rubberduck.Core/UI/CodeExplorer/Commands/ImportCommand.cs

```diff
@@ -106,6 +123,25 @@ private IVBProject TargetProjectFromVbe()
                 : null;
         }
 
+        private IDictionary<ComponentType, IRequiredBinaryFilesFromFileNameExtractor> BinaryFileExtractors(IEnumerable<IRequiredBinaryFilesFromFileNameExtractor> extractors)
+        {
+            var dict = new Dictionary<ComponentType, IRequiredBinaryFilesFromFileNameExtractor>();
+            foreach (var extractor in extractors)
+            {
+                foreach (var componentType in extractor.SupportedComponentTypes)
```

Comment:
if an extractor can support multiple componentTypes, overlaps seem very much possible. This code does not account for that (or rather explicitly only uses the first extractor). A clarifying comment as to why that is sensible might be in order here.

Reply:
I changed it to use all extractors. This should not be too much of a performance problem.